dotCMS/src/main/java/com/dotmarketing/business/RelationshipAPIImpl.java

```diff
@@ -174,24 +174,12 @@ public int maxSortOrder(String parentInode, String relationType) {
         return super.dbRelatedContent(relationship, contentlet, hasParent, live);
     }
 
-    @CloseDBIfOpened
```

Comment:
why did you remove these methods, we shouldn't change the API's

Reply:
Those methods were removed because:
1) The implementation contained outdated queries over tables and fields that do not exist anymore. If a client were using them, for sure an issue had already been reported

2) Those method were not declared in the interface RelationshipAPI

Reply:
oh ok make sense, we shouldn't remove API method for back compatibility however this case is ok!